### DISCUSSION TITLE
Broken CSS on overriden layouts

### DISCUSSION THREAD

**Discussion by cyetain** [imfing/hextra#393] [Q&A]
When I try to over-ride a template or a partial, the CSS breaks for that component?

For instance if I copy footer.html from the theme and place it in layouts/partials/  ... the footer renders without any CSS?

--jabe

**Comment by imfing** [comment]
Are you using version 0.7.x? If so, you might need to copy the code from the 0.7.x version tag, as there have been some changes in later versions.

https://github.com/imfing/hextra/tree/v0.7.3
